crates/bevy_ui/Cargo.toml

```diff
@@ -33,4 +33,4 @@ bevy_utils = { path = "../bevy_utils", version = "0.5.0" }
 # other
 stretch = "0.3"
 serde = {version = "1", features = ["derive"]}
-smallvec = "1.4"
\ No newline at end of file
+smallvec = { version = "1.6", feautres = ["union", "const_generics"] }
```

Comment:
typo found: `feautres` => `features`.

cargo is complaining about this "unused manifest key."